source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+owner_slack: "#modernisation-platform"
+title: Set Up Macie
+last_reviewed_on: 2024-04-18
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+# Introduction
+
+It is relatively easy to set up Macie for users and it need to be done on an account by account basis. **NOTE** Macie ONLY looks at S3 buckets and checks their access information so it will not look at databases or EC2 settings, for example.
+
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one where you want to use macie. There is an AWS video explaining the set up and use of macie [demo](https://www.youtube.com/watch?v=8piwEQJJXdo).
+
+AWS also have a brief outline of the purpose of [Macie](https://docs.aws.amazon.com/macie/latest/user/what-is-macie.html) which can be checked to see if this is suitable.
+
+Starting Macie with Terraform is possible and the instructions are shown below. I have also listed the details of setting up jobs. it may be beneficial to set up jobs manually due to the items that can be included. The example only shows a few of the many items that can be included.
+
+# Terraform code
+
+Below there is an example of how the items were set up in both Example and Cooker following the instructions listed here.
+
+## Example Terraform code
+
+There is one **pre-requisite**, for the code shown here, to allow this to work and run successfully. Make sure that the role for **MemberInfrastructureAccess has ace "macie2:*"** set. If this has not been set up correctly (by the Modernisation Platform Team) the terraform listed below will not work. If this is amended in the future, so that it is enabled globally, this comment will be removed.
+
+Another item is needed before the code below is run. This is the requirements to set up some data. It can be added to "platform_data.tf" or to "data.tf". This shown below in the code (based on example from example and cooker) and is needed to allow the likes of data.aws_s3_bucket.bucket1.id to work. I have shown bucket1 in the code below. 
+
+Full details of this creation, including information on the likes of job type, are shown in the code (macie.tf) in the modernisation-platform-environments repo for example and cooker. The appropriate environment account reference should be used as shown below. If this data value isn't in place look on the code to see how it's set up. You may have something more appropriate set up. The comments can be removed but are included here to provide explanation. They are not shown in the example and cooker repositories.
```

Comment:
You can remove mention of cooker in this as it's one of our test accounts and you can link to the example code.

From `The appropriate environment...` etc onwards isn't needed and adds confusion, it can be removed.

Reply:
Removed